test/recipes/80-test_ssl_new.t

```diff
@@ -70,6 +70,7 @@ my %conf_dependent_tests = (
   "07-dtls-protocol-version.cnf" => !$is_default_dtls || !disabled("sctp"),
   "10-resumption.cnf" => !$is_default_tls || $no_ec,
   "11-dtls_resumption.cnf" => !$is_default_dtls || !disabled("sctp"),
+  "13-fragmentation.cnf" => alldisabled("ec", "dh") || disabled("tls1_3") || disabled("tls1_2"),
```

Comment:
Why disabled these tests when no EC and no DH  ?
The test certificates are EC or DH ones ?

Reply:
no the test is not disabled, it is just re-configured when EC and DH are both disabled.
Some test fail in a different way if that condition is fulfilled.

Reply:
ok